amy/workshops/models.py

```diff
@@ -988,6 +987,13 @@ def archive(self) -> None:
         self.is_active = False
         self.occupation = ""
         self.orcid = ""
+        self.secondary_email = ""
+        self.gender = GenderMixin.UNDISCLOSED
+        self.gender_other = ""
+        # TODO(lb): Remove references to agreements when Consents project is launched
```

Comment:
Is there a story / ticket for this so that we don't forget?

Reply:
So I think it's covered in a broad one here: https://github.com/carpentries/amy/issues/1881 Let me know if you prefer something more detailed. But I think this naturally gets removed when this ticket is done.